doc/about.rst

```diff
@@ -93,7 +93,34 @@ Emeritus Maintainers Team
 The following people have been active contributors in the past, but are no
 longer active in the project:
 
-.. include:: maintainers_emeritus.rst
+.. raw:: html
```

Comment:
the error is coming from here, since the link to `maintainers_emeritus.rst` is being removed. I rather keep the link though, to have one place to keep that information. The same way for other sections like the communication team section.

Reply:
Another option is to remove the new HTML table for the emeritus contributors in this PR, and try it in a new PR, so it's not holding this up. I did make a lot of updates in various sections, so breaking up the PR is one way to move forward.